docs/source/user_guide/training/metric.rst

```diff
@@ -65,46 +59,48 @@ Example:
 
+.. _distortion-metrics:
+
 Distortion metrics
 ------------------
+Distortion metrics are used to measure the difference between the original ``x`` and the reconstructed image ``x_net``.
 
-We implement popular distortion metrics
-(see `The Perception-Distortion Tradeoff <https://openaccess.thecvf.com/content_cvpr_2018/papers/Blau_The_Perception-Distortion_Tradeoff_CVPR_2018_paper.pdf>`_
-for an explanation of distortion vs perceptual metrics):
+.. list-table:: Distortion Metrics
+   :header-rows: 1
 
-.. autosummary::
-   :toctree: stubs
-   :template: myclass_template.rst
-   :nosignatures:
+   * - **Metric**
+     - **Definition**
 
-        deepinv.loss.metric.MSE
-        deepinv.loss.metric.NMSE
-        deepinv.loss.metric.MAE
-        deepinv.loss.metric.PSNR
-        deepinv.loss.metric.SSIM
-        deepinv.loss.metric.QNR
-        deepinv.loss.metric.L1L2
+   * - :class:`deepinv.loss.metric.MSE`
+     - :math:`\text{MSE}(x, \hat{x}) = \frac{1}{n} \sum_{i=1}^n (x_i - \hat{x}_i)^2`
 
-Perceptual metrics
-------------------
+   * - :class:`deepinv.loss.metric.NMSE`
+     - :math:`\text{NMSE}(\hat{x},x) = \frac{\| x - \hat{x} \|_2^2}{\| x \|_2^2}`
+
+   * - :class:`deepinv.loss.metric.MAE`
+     - :math:`\text{MAE}(\hat{x},x) = \frac{1}{n} \sum_{i=1}^n |x_i - \hat{x}_i|`
+
+   * - :class:`deepinv.loss.metric.PSNR`
+     - :math:`\text{PSNR}(\hat{x},x) = 10 \cdot \log_{10} \left( \frac{\text{MAX}^2}{\text{MSE}(\hat{x},x)} \right)`, where :math:`\text{MAX}` is the maximum possible pixel value of the image
+
+   * - :class:`deepinv.loss.metric.SSIM`
+     - :math:`\text{SSIM}(\hat{x},x) = \frac{(2 \mu_x \mu_{\hat{x}} + C_1)(2 \sigma_{x\hat{x}} + C_2)}{(\mu_x^2 + \mu_{\hat{x}}^2 + C_1)(\sigma_x^2 + \sigma_{\hat{x}}^2 + C_2)}`, where :math:`\mu` and :math:`\sigma` are mean and variance
 
-We implement popular perceptual metrics:
+   * - :class:`deepinv.loss.metric.QNR`
+     - :math:`\text{QNR}(\hat{x}) = (1-D_\lambda)^\alpha(1 - D_s)^\beta`, where :math:`D_\lambda` and :math:`D_s` are spectral and spatial distortions
 
-.. autosummary::
-   :toctree: stubs
-   :template: myclass_template.rst
-   :nosignatures:
-    
-        deepinv.loss.metric.LPIPS
-        deepinv.loss.metric.NIQE
+   * - :class:`deepinv.loss.metric.L1L2`
+     - :math:`\text{L1L2}(\hat{x},x) = \alpha \|x - \hat{x}\|_1 + (1 - \alpha) \|x - \hat{x}\|_2`, where :math:`\alpha` is a balancing parameter
 
-Utils
--------
-A set of popular distances that can be used by the supervised and self-supervised losses.
+   * - :class:`deepinv.loss.metric.LpNorm`
+     - :math:`\text{LpNorm}(\hat{x},x) = \|x - \hat{x}\|_p^p`
+
+
+.. _perceptual-metrics:
+
+Perceptual metrics
+------------------
 
-.. autosummary::
-   :toctree: stubs
-   :template: myclass_template.rst
-   :nosignatures:
+We implement no-reference perceptual metrics, they only require the reconstructed image ``x_net``.
 
-    deepinv.loss.metric.LpNorm
\ No newline at end of file
+- :class:`deepinv.loss.metric.NIQE`
```

Comment:
Why remove LPIPS? (note LPIPS isn't no-reference)

Reply:
I've added, sorry for missing this in the previous round

Reply:
I've also changed 'distortion' for 'full reference' and 'perceptual' to 'no reference' to avoid confusions